res/values/strings.xml

```diff
@@ -882,6 +894,20 @@
     <string name="show_window">Show Window</string>
     <string name="login_socks5_experimental_warning">SOCKS5 support is currently experimental. Please use at your own risk. If you type in an address in the e-mail field, there will be DNS lookup that won\'t get tunneled through SOCKS5.</string>
 
+    <!-- desktop keybindings -->
+    <string name="desktop_keybindings_composer">MessageInput / Composer</string>
+    <string name="desktop_keybindings_global">Global Shortcuts</string>
+    <string name="desktop_keybindings_action_switch_between_chats">Switch between Chats</string>
+    <string name="desktop_keybindings_action_scroll_messages">Scroll Messages</string>
+    <string name="desktop_keybindings_action_jump_to_search">Jump to Search</string>
+    <string name="desktop_keybindings_action_focus_composer">Focus Composer</string>
+    <string name="desktop_keybindings_action_open_help">Open Help</string>
+    <string name="desktop_keybindings_action_open_settings">Open Settings</string>
+    <string name="desktop_keybindings_action_force_refresh_network">Force Refresh Network</string>
+    <string name="desktop_keybindings_action_send_message">Send Message</string>
+    <string name="desktop_keybindings_action_insert_newline">Insert Newline</string>
+    <string name="desktop_keybindings_dialog_title">Keybindings</string>
```

Comment:
same here, just call it keybindings - might be, we can resuse these strings when it comes to tablets :)